platform_specific/ubuntu/YamlParameters.cpp

```diff
@@ -0,0 +1,59 @@
+#include "YamlParameters.hpp"
+
+namespace fs = std::filesystem;
+
+extern uint8_t flash_memory;
```

Comment:
I have a warning because of this line:

```
libparams/platform_specific/ubuntu/YamlParameters.cpp:10:46: error: declaration of ‘flash_memory’ shadows a global declaration
```

We can just remove this line because it is unused.